### ISSUE TITLE
Using youtube-dl with OpenDNS

### ISSUE THREAD

**Issue by kiddthekidd** [rg3/youtube-dl#5424]
Hi, thank you for this great tool, its been a ease downloading youtube videos with this.

Now to my issue;

I have set up OpenDNS on my network to filter some things, so my computer is safe for work. One of the things that is filtered is YouTube.com, which means the access is restricted.

I used youtube-dl last week and it easily downloaded the playlist I requested, but now today i got this error(and keep in my mind I just updated my youtube-dl):

"Unable to download webpage: HTTP Error 403: Forbidden (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output."

I tried using youtube-dl with another WiFi-connection and there it seemed to work.

So my question is, how I can make it possible to download via youtube-dl even tho OpenDNS is filtering the youtube domain? It worked first time, so must be possible to make it work again.

full log with --verbose

sudo youtube-dl https://www.youtube.com/watch?v=45ZbDgACTwE --verbose
Password:
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'https://www.youtube.com/watch?v=45ZbDgACTwE', u'--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.04.09
[debug] Python version 2.7.6 - Darwin-14.1.0-x86_64-i386-64bit
[debug] exe versions: none
[debug] Proxy map: {}
[youtube] 45ZbDgACTwE: Downloading webpage
ERROR: Unable to download webpage: HTTP Error 403: Forbidden (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 314, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1673, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)

- Thanks on advance.

**Comment by dstftw** [comment]
Obviously, to make youtube-dl work again you need to unblock youtube domain or anyhow make it resolvable. I'm closing the issue as it's out of scope of youtube-dl.
